src/treq/testing.py

```diff
@@ -239,7 +239,17 @@ def __init__(self, resource):
                 function = _reject_files(function)
 
             setattr(self, function_name, function)
-        self.flush = _agent.flush
+        self.flush = self._agent.flush
+
+    def cleanSessions(self):
+        """
+        Clean up sessions to prevent leaving behind a dirty reactor.
+        If you are using :obj:`StubTreq` with :obj:`twisted.web.server.Session`
+        objects, you most likely have to call this method once you are done,
+        for example during the tearDown of a unittest TestCase.
```

Comment:
For stuff like this using `self.addCleanup` is almost always superior.

Reply:
@twm Are you suggesting replacing this whole method to somehow use said function or to change the documentation?

Reply:
To change the documentation, but it's a moot point if this method is going away.